samples/openapi3/client/petstore/typescript/tests/object_params/package.json

```diff
@@ -15,11 +15,10 @@
     "@types/chai": "4.0.1 - 4.1.x",
     "@types/isomorphic-fetch": "0.0.34",
     "@types/mocha": "^2.2.41",
-    "@types/node": "8.10.38 - 8.10.62",
+    "@types/node": "^22",
```

Comment:
ditto

Reply:
I think there were some type conflicts with the old version. 22 is the latest stable node version.